nautobot/docs/user-guide/platform-functionality/template-filters.md

```diff
@@ -216,12 +216,86 @@ Render a dictionary as formatted JSON.
 
 ### render_markdown
 
-Render text as Markdown.
+Render and sanitize Markdown text into HTML. A limited subset of HTML tags and attributes are permitted in the text as well; non-permitted HTML will be stripped from the output for security.
```

Comment:
```suggestion
Render and sanitize Markdown text into HTML. A limited subset of HTML tags and attributes are permitted in the text; non-permitted HTML will be stripped from the output for security.
```

Reply:
Reads more accurately/completely to me as is - but I'm open to convincing?

Reply:
I was thinking "as well as Markdown".